cypress/integration/85-cv/slos/CreateSLO.spec.ts

```diff
@@ -131,38 +144,41 @@ describe('Create SLO', () => {
     cy.get('input[name="objectiveComparator"]').should('have.value', '<=')
     cy.get('input[name="SLIMissingDataType"]').should('have.value', 'Good')
 
-    cy.contains('span', 'Continue').click()
+    cy.contains('span', 'Continue').click({ force: true })
     cy.contains('h2', 'Set the SLO time window and target').should('be.visible')
 
     cy.get('input[name="periodType"]').should('have.value', 'Rolling')
     cy.get('input[name="periodLength"]').should('have.value', '30')
     cy.get('input[name="SLOTargetPercentage"]').should('have.value', '90')
 
-    cy.fillField('SLOTargetPercentage', '95')
+    cy.get('input[name="periodLength"]').click()
+    cy.contains('p', '28').click({ force: true })
 
-    cy.contains('span', 'Save').click()
+    cy.contains('span', 'Save').click({ force: true })
 
     cy.contains('h2', 'Review changes').should('be.visible')
 
-    cy.contains('span', 'OK').click()
+    cy.intercept('GET', listSLOsCall, getSLODashboardWidgetsAfterEdit)
 
-    cy.intercept('GET', listSLOsCall, { fixture: 'cv/slo/getSLODashboardWidgetsAfterEdit' })
+    cy.contains('span', 'OK').click()
 
+    cy.wait(2000)
```

Comment:
can we wait on API call rather than time ?

```suggestion
      cy.intercept('GET', listSLOsCall, getSLODashboardWidgetsAfterEdit).as('getMessages')
      cy.wait('@getMessages')
```

Reply:
It's getting timeout before rendering `SLO recalculation in progress`. So waiting after `OK` to make API gets completed.